jerry-core/ecma/builtin-objects/ecma-builtin-boolean-prototype.c

```diff
@@ -105,6 +91,44 @@ ecma_builtin_boolean_prototype_object_value_of (ecma_value_t this_arg) /**< this
   return ecma_raise_type_error (ECMA_ERR_MSG ("Argument 'this' is not a Boolean object."));
 } /* ecma_builtin_boolean_prototype_object_value_of */
 
+/**
+ * Dispatcher of the built-in's routines
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+ecma_value_t
+ecma_builtin_boolean_prototype_dispatch_routine (uint16_t builtin_routine_id, /**< built-in wide routine
+                                                                               *   identifier */
+                                                 ecma_value_t this_arg, /**< 'this' argument value */
+                                                 const ecma_value_t arguments_list_p[], /**< list of arguments
+                                                                                         *   passed to routine */
+                                                 uint32_t arguments_number) /**< length of arguments' list */
+{
+  JERRY_UNUSED (arguments_number);
+  JERRY_UNUSED (arguments_list_p);
+
+  ecma_value_t value_of_ret = ecma_builtin_boolean_prototype_object_value_of (this_arg);
+  if (builtin_routine_id == ECMA_BOOLEAN_PROTOTYPE_ROUTINE_VALUE_OF)
+  {
+    return value_of_ret;
+  }
+
+  if (ECMA_IS_VALUE_ERROR (value_of_ret))
```

Comment:
There should be an assert before this if.
`JERRY_ASSERT (builtin_routine_id == ECMA_BOOLEAN_PROTOTYPE_ROUTINE_TO_STRING);`